other/materials_designer/specific_examples/perturbation_ripple_graphene.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
<p>Ripple perturbation of a graphene sheet</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Renamed in docs as well